include/libhashkit-1.0/hashkit.h

```diff
@@ -49,7 +51,9 @@ struct hashkit_st {
     bool is_allocated : 1;
   } options;
 
-  void *_key;
+  bool use_encryption;
+  EVP_CIPHER_CTX *encryption_context;
+  EVP_CIPHER_CTX *decryption_context;
 };
```

Comment:
This breaks the ABI, so either allocate a private struct and reuse `_key`, or this may only be targeted at v2.0